tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -474,3 +474,28 @@ Given('the user has created the following files', function (entryList) {
   })
   return client
 })
+
+When('the user copies permalink to the current folder', function () {
+  return client.page.filesPage().copyPermalink()
+})
+Then('the clipboard content should match current folder permalink', function () {
+  return client.execute(function () {
+    const myInput = document.createElement('INPUT')
+    const inputId = document.createAttribute('id')
+    inputId.value = 'helperClipboardInput'
+    myInput.setAttributeNode(inputId)
+    myInput.setAttribute('style', 'position: absolute; left:0; top:0;') // make sure its visible
+    document.body.appendChild(myInput)
+  })
+    .setValue('#helperClipboardInput', '') // just to focus the element
+    .keys([client.Keys.CONTROL, 'v']) // copy the content of the clipboard into that field
+    .getValue('#helperClipboardInput', function (clipboard) {
+      console.log('clipboardValue ' + clipboard.value)
+      const clipboardContent = clipboard.value
+      webdav.getProperties('/', client.globals.currentUser, ['oc:privatelink'])
```

Comment:
this returns multiple privatelinks, which one to compare against?

Reply:
as soon as we did a propfind on a single node, it will return an array containing a single element.

Reply:
It was related to children props. Depth set to zero In the last commit so there is only one unique propname in response.